src/languages/en-US.ts

```diff
@@ -3484,6 +3522,38 @@ export default class extends Language {
 		COMMAND_STAR_TOPRECEIVERS: 'Top Star Receivers',
 		COMMAND_STAR_TOPRECEIVERS_DESCRIPTION: (medal, id, stars) => `${medal}: <@${id}> (${stars} ${stars === 1 ? 'star' : 'stars'})`,
 
+		/**
+		 * ##################
+		 * SUGGESTIONS COMMANDS
+		 */
+		COMMAND_SUGGEST_NOSETUP: username => `I'm sorry ${username}, but a suggestions channel hasn't been setup.`,
+		COMMAND_SUGGEST_NOSETUP_ASK: username => `I'm sorry ${username}, but a suggestions channel hasn't been setup. Would you like to set-up a channel now?`,
+		COMMAND_SUGGEST_NOSETUP_ABORT: 'Alright then. Aborted creating a new suggestion.',
+		COMMAND_SUGGEST_CHANNEL_PROMPT: 'Please mention the channel you want to set as the suggestions channel. You have 30 seconds.',
+		COMMAND_SUGGEST_CHANNEL_INVALID: 'Didn\'t receive a valid channel mention. Aborting...',
+		COMMAND_SUGGEST_TITLE: id => `Suggestion #${id}`,
+		COMMAND_SUGGEST_SUCCESS: channel => `Thank you for your suggestion! It has been successfully posted to the ${channel} channel`,
+
+		COMMAND_RESOLVESUGGESTION_INVALID_ID: 'That\'s not a valid suggestion ID!',
+		COMMAND_RESOLVESUGGESTION_MESSAGE_NOT_FOUND: 'Couldn\'t find the suggestion\'s message. To prevent clutter, I have removed it from the database.',
```

Comment:
End-users be like: what is a database?

Avoid using technical words.

Reply:
and... replace it with what. 

The other possibility would maybe be `index`, but that's worse.

Reply:
Users don't need to know that the data has been deleted from the database, but that the suggestion itself was deleted.

Therefore something like the following would be more fitting and friendlier:

```suggestion
		COMMAND_RESOLVESUGGESTION_MESSAGE_NOT_FOUND: `${REDCROSS} I was not able to retrieve the suggestion as its message has been deleted.`,
```